### PR TITLE
Nemaniarjun/many commits

### PR DESCRIPTION
**Pull request by nemaniarjun**

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/gui/gallery.py [hunk 1]
```diff
@@ -62,3 +62,23 @@ def blur_image(self):
         """Apply blur to the chosen image."""
         show = self.image.blur(self.slider.get())
         self.show_image(show)
+
+    def remove(self, object):
+        """Remove object from gallery if exists."""
+        if object:
+        object.pack_forget()
```
Comment (nkprince007) [inline uiplib/gui/gallery.py]: Is the indentation correct? :thinking:

Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: It was written in an unnecessary hurry. 
Keep on reviewing. After @lohani2280 's commit is merged I will fix all the gaps and polish it properly.

**Comment by abhsag24** [comment]
Improve commit message in Remove unused code, unneeded slash

File: uiplib/gui/mainGui.py [hunk 2]
```diff
@@ -1,16 +1,18 @@
 """Module that builds the Graphical User Interface."""
 
 import os
-from queue import Queue
+from shutil import copy
 from tkinter import *
 from tkinter.ttk import *
 from tkinter import messagebox
+from threading import Thread, active_count
+from shutil import copy
```
Comment (lohani2280) [inline uiplib/gui/mainGui.py]: you already used `from shutil import copy` in line 4 . why again?

Comment (nemaniarjun) [inline uiplib/gui/mainGui.py]: Ahh.. Copy Paste error. Will fix.

Comment (nemaniarjun) [inline uiplib/gui/mainGui.py]: It was a commit problem. Was adding it in both commits.

**Pull request merged**